apis/opts/mountpoint_test.go

```diff
@@ -20,15 +20,60 @@ func TestCheckBind(t *testing.T) {
 	}
 
 	parseds := []parsed{
-		{bind: "volume-test:/mnt", len: 2, err: false, expectErr: fmt.Errorf("")},
-		{bind: "volume-test:/mnt:rw", len: 3, err: false, expectErr: fmt.Errorf("")},
-		{bind: "/mnt", len: 1, err: false, expectErr: fmt.Errorf("")},
-		{bind: ":/mnt:rw", len: 3, err: false, expectErr: fmt.Errorf(":/mnt:rw")},
-		{bind: "volume-test:/mnt:/mnt:rw", len: 4, err: true, expectErr: fmt.Errorf("unknown volume bind: volume-test:/mnt:/mnt:rw")},
-		{bind: "", len: 0, err: true, expectErr: fmt.Errorf("unknown volume bind: ")},
-		{bind: "volume-test::rw", len: 3, err: true, expectErr: fmt.Errorf("unknown volume bind: volume-test::rw")},
-		{bind: "volume-test", len: 1, err: true, expectErr: fmt.Errorf("invalid bind path: volume-test")},
-		{bind: ":mnt:rw", len: 3, err: true, expectErr: fmt.Errorf("invalid bind path: mnt")},
+		{
```

Comment:
why should we change one line to this structure? just caurious @chuanchang

Reply:
Actually I think the new way is much more readable. 😄